File: examples/compare/steps.py
from decorators import *

@Given(r"I'm a cucumber$")
def _():
   print '.'
   pending()

@Given(r'failing')
def _():
   1 / 0

@Given(r'passing')
def _():
  print '.'
 
@Before
def _():
   print 'before'
 
@After
def _():
   print 'after'

File: examples/compare/steps.rb
Given /^I'm a cucumber$/ do
#   puts '.'
#   @runs ||= 0
#   @runs += 1
#   if @runs > 1
      puts '.'
      pending
#   end
end

Given /^passing$/ do
  puts '.'
end

Given /^failing$/ do
  1 / 0
end

Before do
  puts :before
end

After do
  puts :after
end
